django/applications/catmaid/control/cropping.py

```diff
@@ -387,7 +391,7 @@ def extract_substack_no_rotation( job ):
         class BB:
             pass
         bb = BB()
-        bb.px_x_min = px_x_min
+        bb.px_x_min = px_x_min # FIXME: This does not work
```

Comment:
Nevermind on this one; had dipped into another language for too long and had forgotten that python lets you create instance variables (rather than just assign them values) on a per-instance level.

Reply:
The easiest way to make mypy happy about this is to adjust the BB class to have its instance variables known in the class; will be updating the PR shortly. Let me know what you think about how it looks.

Reply:
Yes, the updated version looks good. In fact, the whole `BB` class definition should be on module level I believe, but no meed to change it if you don't want to change other things as well.